examples/Constraints_Continuous/interpoint.py

```diff
@@ -0,0 +1,264 @@
+
+SMOKE_TEST = "SMOKE_TEST" in os.environ
+BATCH_SIZE = 3
+N_ITERATIONS = 4 if SMOKE_TEST else 15
+TOLERANCE = 0.01
+
+
+set_random_seed(1337)
+
+# ## Defining the Chemical Optimization Problem
+
+# We'll optimize a synthetic chemical reaction with the following experimental parameters:
+# - **Solvent Volume** (10-30 mL per experiment): The amount of solvent used
+# - **Reactant A Concentration** (0.1-2.0 g/L): Primary reactant concentration
+# - **Catalyst Loading** (1-10 mol%): Catalyst amount as percentage of limiting reagent
+# - **Temperature** (60-120 °C): Reaction temperature
+# Note that these ranges are chosen arbitrary and do not represent a specific real-world reaction.
+
+parameters = [
+    NumericalContinuousParameter(
+        name="Solvent_Volume", bounds=(10.0, 30.0), metadata={"unit": "mL"}
+    ),
+    NumericalContinuousParameter(
+        name="Reactant_A_Conc", bounds=(0.1, 2.0), metadata={"unit": "g/L"}
+    ),
+    NumericalContinuousParameter(
+        name="Catalyst_Loading", bounds=(1.0, 20.0), metadata={"unit": "mol%"}
+    ),
+    NumericalContinuousParameter(
+        name="Temperature", bounds=(60.0, 120.0), metadata={"unit": "°C"}
+    ),
+]
+
+# ## Constraint Definition
+
+# We define both intrapoint and interpoint constraints to demonstrate the difference:
+#
+# **Intrapoint constraint** (applied to each individual experiment):
+# - Reagent efficiency: For each experiment, solvent volume must be at least 5 times
+#   the reactant concentration (to ensure proper dilution)
+#
+# **Interpoint constraints** (applied across the entire batch):
+# 1. **Catalyst constraint**: Total catalyst loading across all experiments must equal exactly 30 mol%
+# 2. **Solvent budget**: Total solvent across batch should be ≤ 60 mL
+
+intrapoint_constraints = [
+    ContinuousLinearConstraint(
+        parameters=["Solvent_Volume", "Reactant_A_Conc"],
+        operator=">=",
+        coefficients=[1, -5],
+        rhs=0.0,
+        interpoint=False,
+    ),
+]
+
+interpoint_constraints = [
+    ContinuousLinearConstraint(
+        parameters=["Catalyst_Loading"],
+        operator="=",
+        coefficients=[1],
+        rhs=30.0,
+        interpoint=True,
+    ),
+    ContinuousLinearConstraint(
+        parameters=["Solvent_Volume"],
+        operator="<=",
+        coefficients=[1],
+        rhs=60.0,
+        interpoint=True,
+    ),
+]
+
+# ## Campaign Setup
+
+# We construct the search space by combining parameters with constraints, then create
+# a campaign targeting maximum reaction yield. The
+# {class}`~baybe.recommenders.pure.bayesian.botorch.BotorchRecommender` with
+# `sequential_continuous=False` is required for interpoint constraints as they
+# operate on batches rather than individual experiments.
+
+searchspace = SearchSpace.from_product(
+    parameters=parameters,
+    constraints=intrapoint_constraints + interpoint_constraints,
+)
+
+target = NumericalTarget(name="Reaction_Yield")
+objective = target.to_objective()
```

Comment:
The target is nowhere accessed again
```suggestion
objective = NumericalTarget(name="Reaction_Yield").to_objective()
```